microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -984,6 +1000,11 @@
                                 "id": "one_liter_of_water_a_day_frequency",
                                 "operator": "equal",
                                 "value": "Regularly (3-5 times/week)"
+                            },
+                            {
```

Comment:
These two comparisons were querying only on people who selected "Regularly (3-5 times/week)," which is counter-intuitive. Adding Daily, with agreement/approval from Se Jin.

Reply:
+1